src/contentmanager.cpp

```diff
@@ -212,42 +211,42 @@ void ContentManager::setCurrentLanguage(QString language)
     }
     m_currentLanguage = language;
     emit(currentLangChanged());
-    emit(remoteParamsChanged());
+    emit(filterParamsChanged());
 }
 
 void ContentManager::setCurrentCategoryFilter(QString category)
 {
     m_categoryFilter = category.toLower();
-    emit(remoteParamsChanged());
+    emit(filterParamsChanged());
 }
 
 #define CATALOG_HOST "library.kiwix.org"
 #define CATALOG_PORT 80
 #define CATALOG_URL "library.kiwix.org"
-void ContentManager::updateRemoteLibrary() {
-    if (m_local)
-        return;
-    QUrlQuery query;
-    if (m_currentLanguage != "*") {
-        query.addQueryItem("lang", m_currentLanguage);
-    }
-    query.addQueryItem("count", QString::number(0));
-    if (m_categoryFilter != "all") {
-        query.addQueryItem("tag", m_categoryFilter);
+void ContentManager::updateLibrary() {
+    if (!m_local) {
```

Comment:
It is better to keep the "early exit" here. 
```cpp
if (m_local) {
    emit(booksChanged());
    return;
}
...
```
This way, all the later code is letf unchanged and we avoid unnecessary diff to review.